### DISCUSSION TITLE
Please help!...I am just stuck with 3 bugs in one of my FabricJS based project.

### DISCUSSION THREAD

**Discussion by AnandiTechnomads** [fabricjs/fabric.js#7102] [Q&A]
1. I am showing object scaling size while decrease/increase object size but this size is hiding behind the other objects in the canvas.
![Screenshot 2021-06-05 at 7 07 05 PM](https://user-images.githubusercontent.com/85401045/120893761-89f2fe80-c632-11eb-869c-d80e9cbd2359.png)

2. When I add one object and add label on it and copy/paste that then I am not getting exact position of label on pasted object.
![Screenshot 2021-06-05 at 7 06 32 PM](https://user-images.githubusercontent.com/85401045/120893769-92e3d000-c632-11eb-9ee1-07f9e40db77e.png)

3. Multiple selection using Shift key brings few objects in backward/forward. 
![Screenshot 2021-06-05 at 7 06 06 PM](https://user-images.githubusercontent.com/85401045/120893780-a2631900-c632-11eb-9d14-86aab5238745.png)

**Comment by asturur** [comment]
please help belongs to discussions Q&A i suppose, not a project issue.
